backends/bash/windows/val-verde-platform-sdk-swift-corelibs-xctest-windows

```diff
@@ -22,7 +18,11 @@ COMMON_CFLAGS="\
     -fms-extensions \
     ${COMMON_CFLAGS} \
 " \
-DISABLE_INSTALL_DEB=TRUE \
+PACKAGE_INSTALL_COMPLETION_CALLBACK=package-install-completion-callback \
+SWIFTCFLAGS="\
```

Comment:
SWIFTCFLAGS line is reintroduced and was removed.

Reply:
removed